uzsu/__init__.py

```diff
@@ -46,52 +46,98 @@
 
 
 import logging
+from lib.model.smartplugin import SmartPlugin
+
 from datetime import datetime, timedelta
 
 from dateutil.rrule import rrulestr
 from dateutil import parser
 
 import lib.orb
 
-class UZSU():
+class UZSU(SmartPlugin):
+    """
+    Main class of the UZSU Plugin. Does all plugin specific stuff and provides
+    the update functions for the items
+    """
+    
+    ALLOW_MULTIINSTANCE = False
+    
+    PLUGIN_VERSION = "1.3.0"
+
 
     _items = {}         # item buffer for all uzsu enabled items
 
-    def __init__(self, smarthome, path=None):
+    def __init__(self, smarthome, path=None, *args, **kwargs):
```

Comment:
Is there any reason why this `args` or `kwargs` arguments were added? AFAIS they are not used somewhere.